docs/docsite/rst/porting_guides/porting_guide_2.10.rst

```diff
@@ -135,6 +135,7 @@ Noteworthy module changes
 * The parameter ``message`` in :ref:`grafana_dashboard <grafana_dashboard_module>` module is renamed to ``commit_message`` since ``message`` is used by Ansible Core engine internally.
 * The parameter ``message`` in :ref:`datadog_monitor <datadog_monitor_module>` module is renamed to ``notification_message`` since ``message`` is used by Ansible Core engine internally.
 * The parameter ``message`` in :ref:`bigpanda <bigpanda_module>` module is renamed to ``deployment_message`` since ``message`` is used by Ansible Core engine internally.
+* Ansible modules most no longer rely on being able to import Python modules from the remote user's home directory. Such dependencies will need to live elsewhere in the Python path.
```

Comment:
```suggestion
* Ansible no longer looks for Python modules in the current working directory (the ``remote_uaer``'s home directory) when an Ansible module is run.  This is to fix becoming an unprivileged user on OpenBSD and to mitigate any attack vector if the current working directory is writable by a malicious user. Install any Python modules needed to run the Ansible modules in a system wide location or another directory which is in the ``remote_user``'s ``PYTHONPATH`` and readable by the ``become_user``
```

@acozine or @samccann  please take a look.

I'm finding it hard to know precisely how this should be phrased.  Things that are relevant to the issue but may or may not need to be emphasized in the print guide entry:
* Fixes an issue on OpenBSD when the become user is unprivileged
* Fixes the potential for abuse (but we do not know of any way to have exploited that... It's more hardening than an actual problem)
* This is a change to running an ansible module on a "remote" machine (remote could be localhost)
  * It does not affect things run in the main ansible process
* If a user relied on this behavior, for instance, by installing boto into ~remote_user/boto then they need to install both somewhere else.
  * The somewhere else is anywhere that python ordinarily looks except the current working directory (which is ~remote_user when ansible runs a module).
  * Typical alternate location are system locations like /usr/lib/pythonX.Y/site-packages, /uar/local/lib/pythonX.Y/dist-packages, ~become_user/.local/lib/pythonX.Y/site-packages.  But i don't know if there last one works on all platforms.
* Maybe if there's a guide already in docs about installing Ansible module dependencies we should point people there? (May want to review such a place to add a note about the current working directory being stripped from pythonpath as well). I don't know if such documentation already exists.

Reply:
I don't see anything offhand that talks about this ability to have python in your `remote_user` directory. What should be in the porting guide are what we need users to change in their behavior. So reading the list above, it seems the suggested edits cover most of this nicely (one spelling error on the first 'remote_user' mention). The only thing I don't see clearly in the suggested edit is the bit about this only being applicable to remote machine (which I take to mean managed node in ansible speak.) So maybe add that tidbit to the snippet.

Reply:
Done.